ResourceStatusSystem/Agent/test/Test_Agent_SiteInspectorAgent.py

```diff
@@ -40,7 +40,10 @@ def test__execute(mocker, sitesToBeCheckedValue):
   """
 
   mocker.patch("DIRAC.ResourceStatusSystem.Agent.SiteInspectorAgent.AgentModule.__init__")
-  mocker.patch("DIRAC.ResourceStatusSystem.Agent.SiteInspectorAgent.AgentModule.am_getOption", side_effect=mockAM)
+  mocker.patch(
+      "DIRAC.ResourceStatusSystem.Agent.SiteInspectorAgent.AgentModule._AgentModule__moduleProperties",
+      side_effect=lambda x, y=None: y, create=True
+  )
```

Comment:
Why these changes?

Reply:
In Python 3 this mock wasn't being applied correctly but I've forgotten why...